lib/spack/spack/all_commands.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""Utility functions relevant to all commands, including Spack internal
```

Comment:
I like `command_loading.py`, and moving `get_module_from()` in (although possibly renamed to, say, `get_command_module_from()` in line with your suggestion for `get_module()`, below.

Reply:
Scratch that, `get_module_from()` can't be moved into `command_loading.py` because it is used by `spack.extensions`, which is also used by `command_loading.py`. More circularity.

Reply:
> get_module_from() can't be moved into command_loading.py because it is used by spack.extensions

Apologies but I just checked out your latest branch and I don't see where get_module_from is called in `extensions.py`. For that matter I think `get_command_module_from` can also be moved here.

Reply:
Per previous comments and suggestions, `get_module_from()` was renamed to `get_command_module_from()` and `get_module()` to `get_command_module()`.

>    `get_module_from()` can't be moved into `command_loading.py` because it is used by `spack.extensions`

Not sure what I was looking at when I wrote that. `get_command_module_from()` has now been moved to `command_loading.py`.